samples/symTiny.c

```diff
@@ -0,0 +1,42 @@
+// --------------------------------------------------------------------------
+// Hello World for Sym-1
+//
+// Uses only getchar, putchar and puts, generating smaller code than printf
+//
+// Wayne Parham
+//
+// wayne@parhamdata.com
+// --------------------------------------------------------------------------
+
+#include <symio.h>;
+
+void main(void) {
+   char c = 0x00;
+   int  d = 0x00;
+   int  l = 0x00;
+
+   puts( "Hello World!\n" );
+
+   puts( "Type a line and press ENTER, please:\n" );
+
+   for( l = 0; l < 2; l++ ) {
+      beep();
+      for( d = 0; d < 10 ; d++ ) {
+      }
+   }
+
+   while( c != '\r' ) {
+      c = getchar();
+      putchar( c );
+   }
+
+   puts( "\n\nThanks!\n" );
+
+   for( l = 0; l < 5; l++ ) {
+      beep();
+      for( d = 0; d < 10 ; d++ ) {
+      }
+   }
+
+   return;
+}
```

Comment:
Please change this code to adhere to the cc65 C code style rules - see https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide

Reply:
**DONE**.  Please check and confirm.